experiments/arctic_simulation.jl

```diff
@@ -77,14 +76,14 @@ dynamics = SeaIceMomentumEquation(grid;
                                   coriolis = ocean.model.coriolis,
                                   top_momentum_stress = (u=τua, v=τva),
                                   bottom_momentum_stress = τo,
-                                  ocean_velocities = (u=SSU, v=SSV),
+                                  ocean_velocities = (u=0.1*SSU, v=0.1*SSV),
```

Comment:
why 0.1 here? or it's just an example, doesn't matter really?
did you just reduce how much the sea ice feels the ocean?

Reply:
yep, this is what they use in SI3 apparently and I am not even sure why ahah. In practice, this is just an example, we should investigate these parameters